### PR TITLE
cilium-health: Fix logging initialization

### PR DESCRIPTION
**Pull request by vadorovsky**
Allow to set up log driver and log driver options for cilium-health
process.

Signed-off-by: Michal Rostecki <mrostecki@suse.de>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5023)

### CODE REVIEW

**Comment by vadorovsky** [comment]
@aanm PTAL

**Comment by vadorovsky** [comment]
test-me-please

File: cilium-health/cmd/root.go [hunk 1]
```diff
@@ -74,13 +76,14 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	flags := rootCmd.PersistentFlags()
 	flags.StringP("admin", "", string(serverPkg.AdminOptionUnix), "Expose resources over 'unix' socket, 'any' socket")
+	flags.StringP("cilium", "c", "", "URI to Cilium server API")
 	flags.BoolP("debug", "D", false, "Enable debug messages")
 	flags.BoolP("daemon", "d", false, "Run as a daemon")
-	flags.BoolP("passive", "p", false, "Only respond to HTTP health checks")
```
Comment (ianvernon) [inline cilium-health/cmd/root.go]: Why are "passive" and "pidfile" removed?

File: Documentation/cmdref/cilium-health_status.md [hunk 2]
```diff
@@ -25,14 +25,14 @@ cilium-health status
...
-  -p, --passive          Only respond to HTTP health checks
-      --pidfile string   Write the PID to the specified file
+      --admin string             Expose resources over 'unix' socket, 'any' socket (default "unix")
+  -c, --cilium string            URI to Cilium server API
+  -d, --daemon                   Run as a daemon
+  -D, --debug                    Enable debug messages
+  -H, --host string              URI to cilium-health server API
+  -i, --interval uint            Interval (in seconds) for periodic connectivity probes (default 60)
+      --log-driver stringSlice   Logging endpoints to use for example syslog, fluentd
+      --log-opt map              Log driver options for cilium (default map[])
```
Comment (ianvernon) [inline Documentation/cmdref/cilium-health_status.md]: cilium --> cilium-health

**Comment by ianvernon** [comment]
Build failure : 
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/5346

Appears related. cilium-health failed to start, and thus the CI tests failed:
```
cilium-health-ep                           never          34s ago      7       Endpoint failed to run: stat /var/run/cilium/state/health-endpoint.pid: no such file or directory
```

**Review by ianvernon — CHANGES_REQUESTED** [review]

File: cilium-health/cmd/root.go (hunk 1 above)
Comment (vadorovsky) [reply cilium-health/cmd/root.go]: Sorry, they were somehow removed during cherry-picking. I will add them again

**Comment by vadorovsky** [comment]
test-me-please

**Comment by vadorovsky** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Review by tgraf — APPROVED** [review]

**Pull request merged**